apps/spruce/docs/decisions/2023-12-13_version_page_logic.md

```diff
@@ -4,6 +4,15 @@
 - date: 2023-12-13
```

Comment:
The status should be deprecated 
https://github.com/joelparkerhenderson/architecture-decision-record/tree/main/locales/en/templates/decision-record-template-by-michael-nygard